pages/team-structure.js

```diff
@@ -0,0 +1,402 @@
+import React, { useState, useEffect } from 'react';
+import styled, {ThemeProvider} from 'styled-components';
+import PageTemplate from '../components/templates/PageTemplate';
+import Card from '../components/atoms/Card';
+import TeamHierFormWrapper from '../components/molecules/TeamHierFormWrapper';
+import Header3 from '../components/atoms/Header3';
+import TeamHierMemberListGrid from '../components/molecules/TeamHierMemberListGrid';
+import NextSetButton from '../components/atoms/NextSetButton';
+
+import theme from "../components/theme";
+
+import { SystemComponent } from '../components/atoms/SystemComponents';
+
+// TODO: slowly move away from using static data
+const director_json = [
+    {
+        _id: 1,
+        name: "Technical Director",
+        leader: {
+            _id: 1,
+            name: {
+                first: "Clive",
+                last: "Chan"
+            },
+        }
+    }
+];
+
+const subteam_json = [
```

Comment:
You might be able to restructure the dummy data to the same as the normalized redux data so that we don't have to change this later?

Reply:
I will definitely do that, but I may not have time before the Nov. 30 deadline. Will add a TODO note in this file.